src/app/csc/csc.component.html

```diff
@@ -65,6 +65,10 @@ <h4 class="no_data">
         class="csc-projects"
         *ngIf="!dataLoading && filteredCscProjectsList.length > 0"
       >
+        <p>
+          <strong>Total projects:</strong>
```

Comment:
Suggest removing the `<strong>` and having it read "Showing (x) matching projects."